src/lib.rs

```diff
@@ -479,6 +479,40 @@ impl<T> Event<T> {
 
         inner
     }
+
+    /// Return the listener count
+    ///
+    /// This is only available when `std` feature is enabled.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use event_listener::Event;
+    ///
+    /// let event = Event::new();
+    ///
+    /// assert_eq!(event.total_listeners().unwrap(), 0);
+    ///
+    /// let listener1 = event.listen();
+    /// assert_eq!(event.total_listeners().unwrap(), 1);    
+    ///
+    /// let listener2 = event.listen();
+    /// assert_eq!(event.total_listeners().unwrap(), 2);        
+    ///
+    /// drop(listener1);
+    /// drop(listener2);
+    /// assert_eq!(event.total_listeners().unwrap(), 0);        
+    ///
+    /// ```
+    #[cfg(feature = "std")]
+    #[inline]
+    pub fn total_listeners(&self) -> Result<usize, String> {
```

Comment:
I would like to avoid using `String` as an error type. If we need to have an error I would introduce a `MutexPoisoned` newtype and use that as the error type. But other places in `event-listener` panic if the `Mutex` is poisoned, so we should continue that pattern.